silx/gui/plot/tools/roi.py

```diff
@@ -272,12 +339,23 @@ def createRoi(self, roiClass, points, label='', index=None):
            number of ROIs has been reached.
         """
         roi = roiClass(parent=None)
-        roi.setName(str(label))
+        if label is not None:
+            roi.setName(str(label))
         roi.setFirstShapePoints(points)
 
         self.addRoi(roi, index)
+        if roi.isSelectable():
+            self.setSelectedRoi(roi)
         return roi
 
+    def containsRoi(self, roi):
```

Comment:
Is this really needed? `roi in manager.getRois()` would do the job.

Reply:
Yes. That avoid a copy, and as it is explicit, when you use it you don't have to think about.

Reply:
Creating a tuple from a list doesn't count much, I doubt we'll have millions of rois...
To me `in` is explicit.